cosmos/providers/dbt/core/profiles/glue/aws_role_arn.py

```diff
@@ -0,0 +1,52 @@
+"Maps Aws connections to dbt profiles."
+from __future__ import annotations
+
+from logging import getLogger
+from typing import Any
+
+from ..base import BaseProfileMapping
+
+logger = getLogger(__name__)
+
+
+class AWSRoleARNProfileMapping(BaseProfileMapping):
+    """
+    Maps Airflow Aws connections to Glue dbt profiles.
+
+    https://docs.getdbt.com/docs/core/connect-data-platform/glue-setup
+    https://airflow.apache.org/docs/apache-airflow-providers-amazon/stable/connections/aws.html
+    https://github.com/aws-samples/dbt-glue
+    """
+
+    airflow_connection_type: str = "aws"
+    is_community: bool = True
+
+    # Reference link:
+    required_fields = [
+        "role_arn",
+        "region",
+        "workers",
+        "worker_type",
+        "schema",
+        "session_provisioning_timeout_in_seconds",
```

Comment:
Can we define some default values instead of expecting the user to define all these arguments? This could apply to `session_provisioning_timeout_in_seconds` (e.g. 120), `workers` (e.g. 1) and `worker_type` (e.g. Standard).

Reply:
I don't think we should provide defaults here - these are the required fields according to the [dbt-glue package](https://github.com/aws-samples/dbt-glue#example-config) (which doesn't provide default values), so for consistency's sake we should do the same.

Reply:
@jlaneve I think it depends if we want to be Airflow-oriented or DBT-oriented.

The DBT profile aims to centralise all the database connection information in a single place. Airflow connections have the same purpose. However, in most cases, the mandatory information in Airflow connections is a subset of the required information in DBT.

If we want to be consistent with DBT required parameters, this will mean that Airflow Cosmos users will have to define "database connection" properties in two places: in the Airflow connection and as an argument of the Cosmos operator, task group or DAG. It feels like we are defying the original purpose of this centralised "connection" representation.

To have default values for DBT profiles required values which are not required by Airflow connections would simplify the usage for those who do not want to customise these values - avoiding having to define mandatory properties in two different places.

@jlaneve, if you think this is good enough for 1.0, we can stick to this pattern of forcing users to define profile properties in multiple places, and we can review this in future  Cosmos versions.

Reply:
@tatiana Understood, so in next version, I think we need Comos profile. It initiate with connection id an custom values.
For each kind of data flatform we will have a class.